README.md

```diff
@@ -63,21 +63,22 @@ Arguments
 Options
 -------
 
-- `--client-max-body-size` 'Max body size for HTTP client (in bytes).
-- `--client-max-header-size` 'Max header size for HTTP client (in bytes).
-- `--client-redirects=5` Set the maximum number of times the client should redirect (`0` to never redirect).
-- `--client-security-level=1` Set the default SSL [security
+- `--client-max-body-size`: 'Max body size for HTTP client (in bytes).
+- `--client-max-header-size`: 'Max header size for HTTP client (in bytes).
+- `--client-redirects=5`: Set the maximum number of times the client should redirect (`0` to never redirect).
+- `--client-security-level=1`: Set the default SSL [security
   level](https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_security_level.html)
-- `--client-timeout=15000` Set the maximum amount of time (in milliseconds)
+- `--client-timeout=15000`: Set the maximum amount of time (in milliseconds)
   the client should wait for a response, defaults to 15,000 (15 seconds).
 - `--concurrency`: Number of simultaneous HTTP requests to use.
-- `--display-bufsize=10` Set the number of URLs to consider when showing the
+- `--display-bufsize=10`: Set the number of URLs to consider when showing the
   display.
-- `--display=+memory` Set, add or remove elements of the runtime display
+- `--display=+memory`: Set, add or remove elements of the runtime display
   (prefix with `-` or `+` to modify the default set).
-- `--exclude-url=logout` (multiple) Exclude URLs matching the given PCRE pattern.
-- `--header="Foo: Bar"` (multiple) Specify custom header(s).
-- `--include-link=foobar.html` Include given link as if it were linked from the
+- `--exclude-url=logout`: (multiple) Exclude URLs matching the given PCRE pattern.
+- `--header="Foo: Bar"`: (multiple) Specify custom header(s).
+- `--help`: See documentation in this file.
```

Comment:
Should be `display application help` I think :)

Reply:
You're right! I updated the patch.